.circleci/config.yml

```diff
@@ -229,7 +229,7 @@ jobs:
           command: inv -e agent.build --flavor iot
       - run:
           name: test iot agent
-          command: DD_HOSTNAME=test-circleci-hostname ./bin/agent/agent -c ./bin/agent/dist check cpu
+          command: mkdir -p /opt/datadog-agent/run && DD_HOSTNAME=test-circleci-hostname ./bin/agent/agent -c ./bin/agent/dist check cpu
```

Comment:
Why do we need to create the directory as part of this command? Should the `inv -e *.build` take care of that?

Reply:
The directory is normally created by the packages, this test is taking shortcuts to run an `agent` command without a full Agent setup.
On my side, I don't think it's appropriate for the `build` command to take care of that.